docs/refguide/api.md

```diff
@@ -262,12 +290,57 @@ Usage: `observe(object, property?, listener, fireImmediately = false)`
 Low-level api that can be used to observe a single observable value.
 [&laquo;details&raquo;](observe.md)
 
-### `useStrict`
-Usage: `useStrict(boolean)`.
-Enables / disables strict mode *globally*.
+### `decorate`
```

Comment:
probably this one should be moved to the observable section in the beginning?

Reply:
yup moved just under decorators